src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -660,13 +660,26 @@
   (let [row-thunk (row-thunk driver rs rsmeta)]
     (qp.reducible/reducible-rows row-thunk canceled-chan)))
 
+(defmulti inject-remark
+  "Injects the remark into the SQL query text."
+  {:arglists '([driver sql remark])}
```

Comment:
It should document the version it was added in.

Reply:
Done, thanks!